tests/subsys/pm/device_runtime/src/dummy_driver.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sys/printk.h>
+#include <zephyr/types.h>
+#include <pm/device_runtime.h>
+#include "dummy_driver.h"
+
+static uint32_t device_power_state;
+
+static int dummy_wait(const struct device *dev)
```

Comment:
because with this I can do

- open(async)
- open(async)
- close(async)
- wait()

If I put it in open/close I will end up having:
- open(async)
- - wait()
- open(async)
- - wait()
- close(async)
-- wait()

And the test will be different.